src/test/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOptionsMapperFactoryTest.java

```diff
@@ -45,7 +58,11 @@ void returnLLPCertificateOptionsMapperWhenCompanyTypeLLP() {
     @Test
     void returnLLPCertificateOptionsMapperWhenCompanyTypeLP() {
         // given
+        when(featureOptions.isLpCertificateOrdersEnabled()).thenReturn(true);
```

Comment:
returnLPCertificateOptionsMapperWhenCompanyTypeLLP

Reply:
LLP changed to LP in method name